bchrpc/server.go

```diff
@@ -1041,15 +1121,30 @@ func (s *GrpcServer) GetUnspentOutput(ctx context.Context, req *pb.GetUnspentOut
 		coinbase = entry.IsCoinBase()
 	}
 
+	var (
+		slpToken      *pb.SlpToken
+		tokenMetadata *pb.TokenMetadata
+	)
+	if s.slpIndex != nil &&
+		req.Index > 0 &&
+		isSlpInMempool &&
+		req.IncludeMempool {
+		slpToken, err = s.getSlpToken(txnHash, req.Index)
+		tokenID, _ := chainhash.NewHash(slpToken.TokenId)
```

Comment:
Error handling is good for your health. =)

Reply:
done